exercises/all-your-base/all-your-base.jl

```diff
@@ -0,0 +1,5 @@
+# Implement number base conversion
+
+function all_your_base(digit_string::Vector, input_base::Integer, output_base::Integer)

```

Comment:
```suggestion
function all_your_base(digit_string, input_base, output_base)
```

I think it's better to leave out the type annotations because they're not needed in this case